.github/workflows/ci.yml

```diff
@@ -177,17 +204,22 @@ jobs:
       - uses: actions/checkout@v2.3.4
         with:
           persist-credentials: false
+
       - uses: actions-rs/toolchain@v1
         with:
           toolchain: stable
           components: rustfmt
           override: true
-      - run: rustup component add rustfmt
+      #- run: rustup component add rustfmt
```

Comment:
It seems like we could just delete this line, since `rustfmt` is in the list of components.